.github/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,77 @@
+
+## Our Standards
+
+Examples of behavior that contributes to creating a positive environment
+include:
+
+* Using welcoming and inclusive language
+* Being respectful of differing viewpoints and experiences
+* Gracefully accepting constructive criticism
+* Focusing on what is best for the community
+* Showing empathy towards other community members
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexualized language or imagery and unwelcome sexual attention or
+ advances
+* Trolling, insulting/derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or electronic
+ address, without explicit permission
+* Other conduct which could reasonably be considered inappropriate in a
+ professional setting
+
+## Our Responsibilities
+
+Project maintainers are responsible for clarifying the standards of acceptable
+behavior and are expected to take appropriate and fair corrective action in
+response to any instances of unacceptable behavior.
+
+Project maintainers have the right and responsibility to remove, edit, or
+reject comments, commits, code, wiki edits, issues, and other contributions
+that are not aligned to this Code of Conduct, or to ban temporarily or
+permanently any contributor for other behaviors that they deem inappropriate,
+threatening, offensive, or harmful.
+
+## Scope
+
+This Code of Conduct applies both within project spaces and in public spaces
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at [contact@ramnes.eu](mailto:contact@ramnes.eu).
+Allcomplaints will be reviewed and investigated and will result in a response that
+is deemed necessary and appropriate to the circumstances. The project team is
+obligated to maintain confidentiality with regard to the reporter of an incident.
+Further details of specific enforcement policies may be posted separately.
+
+Project maintainers who do not follow or enforce the Code of Conduct in good
+faith may face temporary or permanent repercussions as determined by other
+members of the project's leadership.
+
+## Attribution
+
+This Code of Conduct is adapted from the [Contributor Covenant][homepage],
+version 1.4,
+available at [here](https://www.contributor-covenant.org/version/1/4/code-of-conduct.html).
+
+[homepage]: https://www.contributor-covenant.org
+
+For answers to common questions about this code of conduct, see
+[FAQ](https://www.contributor-covenant.org/faq).
```

Comment:
A CoC is outside the scope of this PR, can we put it aside for now?

Reply:
Definitely, if you say. But I included it because it will be present in the docs site. Most projects generally have it. So, I thought why not.

In the site, it will be like this https://aahnik.github.io/notion-sdk-py/contributing/code_of_conduct/

If I remove Coc, a ripple effect will be caused. We need to remove it from the nav bar, as well as generate.py

May be, we could edit the pr to the title of "Documentation Site" ?

This project will be used by huge no of people one day. It will be as popular as discord.py, telethon, and other pythonic API wrappers. So, I think having a full blown docs site is great.

Reply:
Yeah, I agree with you that it's important to have one, but a CoC is a serious opinion statement, and I'm not feeling confortable with the contributor covenant. Since I've never had to choose or write one, I don't want to rush this and need more time to think. I wouldn't want the rest of this PR to be delayed though, and that's why I'd prefer that we put it aside for now. :)

Reply:
allright!